### PR TITLE
 Fix read me link

### PR DESCRIPTION
**Pull request by akenzua**
[TP12729](https://maps.tpondemand.com/restui/board.aspx?#page=board/5175634625349962976&appConfig=eyJhY2lkIjoiREI5QTI2RDFCNTJDN0JFNTQxMDRDRDIyN0ZFNDIzRDYifQ==&boardPopup=bug/12729/silent)

This PR add the correct link to the read me on result page of mortgage affordability calculator

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">It looks like this is a Rails engine, but no changes to <a href="https://github.com/moneyadviceservice/mortgage_calculator/blob/e73013c6595c99a711533781e5bf10bbe1302677/lib/mortgage_calculator/version.rb">lib/mortgage_calculator/version.rb</a> detected. Did you forget to bump the version?</td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by davidtrussler — DISMISSED** [review]
If possible can we delete the file `spec/dummy/log/development.log` from the PR?

@akenzua There are some failing tests that need to be fixed before approval. These look to me like the ones that were already fixed relating to the addition of the completion date on the tool. Can we ensure that those are in place and we have not reverted to an earlier version of the tool?

**Review by davidtrussler — APPROVED** [review]
This is fine: the failing tests are not caused by the changes here so can be dealt with separately. If possible we should delete the file `spec/dummy/log/development.log` from the commit. Otherwise this is good to go. 👍

**Pull request closed without merging**
